Provisioning/Vault/cert-manager-demo.tf

```diff
@@ -0,0 +1,78 @@
+# This is an example for a demo app with Ingress for demonstrating cert-manager
+# + Vault PKI/ACME capabilities
+
+locals {
+  manifests = fileset("./manifests/cert-manager-demo", "*.yaml")
+}
+
+resource "kubernetes_manifest" "vault-cert-manager-demo" {
+  for_each = local.manifests
+  manifest = yamldecode(file("./manifests/cert-manager-demo/${each.value}"))
+}
+
+# Prepare PKI for ACME
+resource "vault_generic_endpoint" "enable-acme" {
+  path                 = "${vault_mount.pki.path}/config/acme"
+  ignore_absent_fields = true
+  disable_delete       = true
+  write_fields         = ["enabled"]
+  data_json            = <<EOT
+{
+  "enabled": "true"
+}
+EOT
+}
+
+resource "vault_generic_endpoint" "config-acme" {
+  path                 = "${vault_mount.pki.path}/config/cluster"
+  ignore_absent_fields = true
+  disable_delete       = true
+  write_fields         = ["path", "aia_path"]
+  data_json            = <<EOT
+{
+  "path": "https://vault.vault.svc.cluster.local:8200/v1/${vault_mount.pki.path}",
```

Comment:
it is crucial that these paths correspond to the server in the cert-manager cluster issuer